mailpoet/lib/Cron/Workers/InactiveSubscribers.php

```diff
@@ -48,7 +54,17 @@ public function processTaskStrategy(ScheduledTaskEntity $task, $timer) {
     // Handle activation/deactivation within interval
     $meta = $task->getMeta();
     $lastSubscriberId = isset($meta['last_subscriber_id']) ? $meta['last_subscriber_id'] : 0;
-    $maxSubscriberId = isset($meta['max_subscriber_id']) ? $meta['max_subscriber_id'] : (int)Subscriber::max('id');
+
+    if (isset($meta['max_subscriber_id'])) {
+      $maxSubscriberId = $meta['max_subscriber_id'];
+    } else {
+      $maxSubscriberId = $this->entityManager->createQueryBuilder()
+        ->select('MAX(s.id)')
+        ->from(SubscriberEntity::class, 's')
+        ->getQuery()
+        ->getSingleScalarResult();
+    }
```

Comment:
Would it be more appropriate to move this code into `\MailPoet\Subscribers\SubscribersRepository` and then `use` just the repository instead of the subscriber entity and entity manager?

I honestly don't know because I haven't worked much on doctrine myself, but it looks like most similar code appears in repository classes.